frontend/src/screens/admin/admin-reports.tsx

```diff
@@ -25,8 +30,41 @@ const LearnerActivity = () => {
                         value={monthsAgo}
                         onChange={setMonthsAgo}
                     />
-                    <Button component='a' target='_blank' href={`${ENV.API_ADDRESS}/api/v1/admin/reports/learner-activity?months_ago=${monthsAgo}`}>
-                        Generate Report
+                    <Button onClick={async () => {
```

Comment:
This logic could probably be extracted out of the JSX and up in the top of the component to keep things cleaner 😄